lv_conf_template.h

```diff
@@ -83,6 +83,13 @@
     #endif
 #endif  /*LV_USE_STDLIB_MALLOC == LV_STDLIB_BUILTIN*/
 
+/*Layer memory allocation limits*/
+    /** Limit the max allocated memory for simple and transformed layers.
```

Comment:
```suggestion
     /* Limit the max allocated memory for simple and transformed layers.
```

What do you mean by this?
> It should be enough to store the largest widget too (width x height x 4 area)